### ISSUE TITLE
Confusing variable in documentation 

### ISSUE THREAD

**Issue by bastienmoulia** [angular-translate/angular-translate#1143]
In [this](http://angular-translate.github.io/docs/#/guide/07_multi-language#multi-language_determining-preferred-language-automatically) part of the documentation there is this exemple:
```
$translateProvider.determinePreferredLanguage(function () {
  var preferredlangKey = '';
  // some custom logic's going on in here
  return preferredLangKey;
});
```

Once the variable is preferred**l**angKey and the other preferred**L**angKey with an uppercase.

**Comment by tspaeth** [comment]
Hi!

Thanks for your report. Well, yes, could be a typo. But the code example isn't complete anyway :-)...

**Comment by knalli** [comment]
Typo fixed in 45ebf09 -- other things: File new ones.
